esp-wifi/src/timer/riscv.rs

```diff
@@ -28,8 +28,17 @@ const TIMER_DELAY: fugit::HertzU32 = fugit::HertzU32::from_raw(crate::CONFIG.tic
 static ALARM0: Mutex<RefCell<Option<Alarm<Periodic, 0>>>> = Mutex::new(RefCell::new(None));
 
 pub fn setup_timer(systimer: TimeBase) {
+    unsafe {
+        riscv::interrupt::disable();
```

Comment:
Is a blanket disable necessary? What re-enables other interrupts?

Reply:
we already have a `riscv::interrupt::disable();` in the next function called (we had that forever)